### ISSUE TITLE
Not specifying an icon gives a spurious "icon" textbox

### ISSUE THREAD

**Issue by jonnysmith1981** [souly1/ng-walkthrough#60]
I am trying to create a walkthrough step *without* an icon, using this HTML:
```
<walkthrough ng-if="vm.sorResults !== null && vm.sorResults.length > 0 && $first" wid="sorFavourite" walkthrough-type="tip" focus-element-selector="#focus-item"
		use-button="true" force-caption-location="BOTTOM" has-backdrop="true" tip-color="WHITE" is-active="vm.showTips" main-caption="You can add SORs to your Favourite list by swiping left on the SOR and clicking 'Favourite'.">	
	</walkthrough>
```

However i am getting a weird textbox with the word "icon" in it appearing quite a way above the walkthrough step

![](https://cloud.githubusercontent.com/assets/8254912/21614145/2fa1950a-d1d0-11e6-8a1e-d2c822bcf872.png)

**Comment by souly1** [comment]
Fixed in v1.0.4
Download and update here if issue resolved.
Thanks

**Comment by souly1** [comment]
Assume closed due to lack of response.
Will reopen if still exists
